chainermn/extensions/multi_node_evaluator.py

```diff
@@ -1,40 +1,36 @@
-def create_multi_node_evaluator(actual_evaluator, communicator):
+import types
+
+
+def create_multi_node_evaluator(evaluator, communicator):
```

Comment:
I agree that `evaluator` might be a better argument name as the name `actual_evaluator` somewhat discloses the implementation of `create_multi_node_evaluator`, which is not good. On the other hand, we want to follow Chainer's API compatibility rule (http://chainer.readthedocs.io/en/stable/compatibility.html) as much as possible. I think this change conflicts the rule. What do you think?